gattc_sd.go

```diff
@@ -31,14 +31,25 @@ var discoveringService struct {
 	endHandle   volatile.Register16
 }
 
+// uuidWrapper is a type alias for UUID so we ensure no conflicts with
+// struct method of the same name.
+type uuidWrapper = UUID
+
 // DeviceService is a BLE service on a connected peripheral device. It is only
 // valid as long as the device remains connected.
 type DeviceService struct {
+	uuidWrapper
```

Comment:
This causes the same 16 byte space to be occupied so this `uuidWrapper` doesn't save anything. I would suggest either using `C.ble_uuid_t` and converting that to a regular `UUID` type when the `UUID` method is called (code for this conversion has not yet been written), or leaving it out for now. The latter would probably be the best.

Reply:
Updated.